src/libraries/Builder/Microsoft.Agents.Builder/App/AdaptiveCards/AdaptiveCardInvokeResponseFactory.cs

```diff
@@ -13,15 +14,25 @@ public static class AdaptiveCardInvokeResponseFactory
         /// <summary>
         /// Returns response with type "application/vnd.microsoft.card.adaptive".
         /// </summary>
-        /// <param name="adaptiveCard">An Adaptive Card.</param>
+        /// <param name="adaptiveCard">An AdaptiveCard instance.</param>
         /// <returns>The response that includes an Adaptive Card that the client should display.</returns>
-        public static AdaptiveCardInvokeResponse AdaptiveCard(AdaptiveCard adaptiveCard)
+        public static AdaptiveCardInvokeResponse AdaptiveCard(global::AdaptiveCards.AdaptiveCard adaptiveCard)
```

Comment:
the global:: is usually a bad sign.. what's going on here?

Reply:
We have both a namespace and type collision.  What we want there is AdaptiveCards.AdaptiveCard (from the AdaptiveCard package).  But what we are in is Microsoft.Agents.Builder.App.**AdapativeCards** and there is an **AdaptiveCard** type.

Is there another way?  Drop that overload?